docs/DeveloperGuide.md

```diff
@@ -171,6 +171,31 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Enhanced Find Logic
+
+The previous implementation find target is limited to name field only. Hence, extending the find function will increase the user experience by enable them easily search `Person` with `phone`
+`email` and `address` field.
+This is done by creating `searchType` model within the `model util` package and extract original `NameCdonstrainsKeywordsPreicate` to form `CdonstrainsKeywordsPreicate` then create different inherit of three different types of predicate.
+
+### Enhanced Command Features
+
+#### Chain Command
+
+This extension allow user to chain multiple commands together, also noted one command fail may lead to following commands invalid (e.g. if create person failed, you cannot edit the newly created person) and exception will throw.
```

Comment:
```suggestion
This extension allows user to chain multiple commands together.

 :information_source: **Note:** A command failure may lead to following commands being invalid (e.g. if create person failed, you cannot edit the newly created person), causing an exception to be thrown.
```

Fix grammar. Suggestion have the note written in the stated format

Reply:
thanks format changed